financial-templates-lib/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -47,9 +47,23 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
       return undefined;
     }
 
+    // Set first price period in `historicalPricePeriods` to first non-null price.
+    let firstPricePeriod;
+    for (let p in this.historicalPricePeriods) {
```

Comment:
@mrice32 this is new logic to handle any stray null prices in `this.historicalPricePeriods`. The price feed needs to know which price period is the "first", but in case `this.historicalPricePeriods[0] === null` it should use the first non-null price as the "first" instead.